Source/DataSources/CzmlDataSource.js

```diff
@@ -1152,6 +1152,7 @@ define([
         processPacketData(Boolean, box, 'outline', boxData.outline, interval, sourceUri, entityCollection);
         processPacketData(Color, box, 'outlineColor', boxData.outlineColor, interval, sourceUri, entityCollection);
         processPacketData(Number, box, 'outlineWidth', boxData.outlineWidth, interval, sourceUri, entityCollection);
+        processPacketData(Number, box, 'shadows', boxData.shadows, interval, sourceUri, entityCollection);
```

Comment:
If this is an enum, it needs to be treated as such in CZML.  Follow the existing pattern of e.g. `HorizontalOrigin`.